concent_api/core/models.py

```diff
@@ -518,3 +519,39 @@ def clean(self):
             raise ValidationError({
                 'pending_response': 'PaymentInfo should be related with Pending Response'
             })
+
+
+class GlobalTransactionState(Model):
+    """
+    Represents state of whole transaction service by storing last `nonce` used.
+
+    There should always be at most one object of this type with id = 0. If it does not exist, it should be created and
+    nonce initialized with a value obtained from SCI (Client.get_transaction_count()).
+    """
+
+    nonce = DecimalField(max_digits=32, decimal_places=0, default=0, unique=True)
+
+    def save(self, *args, **kwargs):  # pylint: disable=arguments-differ
+        self.pk = 0
+        super().save(*args, **kwargs)
+
+
+class PendingEthereumTransaction(Model):
+    """
+    Represents pending Ethereum transaction state.
+    """
+
+    nonce = DecimalField(max_digits=32, decimal_places=0, unique=True)
+
+    gasprice = CharField(max_length=32)
+    startgas = CharField(max_length=32)
+    value = CharField(max_length=32)
+
+    to = BinaryField(max_length=20)
+    data = BinaryField()
+
+    v = IntegerField()
+    r = CharField(max_length=32)
```

Comment:
Field `r` in `transaction.json` is stored as a integer. In my tests i had `r` field which equaled 105276041803796697890139158600495981346175539693000174052040367753737207356915

Reply:
I have checked that even `BigIntegerField` can store numbers in range `from -9223372036854775808 to 9223372036854775807` so it is too small for our needs. I think the best would be to convert this number to string and then store as `TextField()`

Reply:
Yep, that what I was already pointing out on daily few days ago, `BigIntegerField` is not enough for this, but `Decimal` could do in this case, as I now know how sample this data will look exactly in `Transaction`.

@kbeker I see you have liked `TextField` very much but believe me it is not a cure for everything ;)

Reply:
Maximum value for a 32-byte unsigned int is just `2**(32*8)-1`. 78 decimal digits is enough.